### PR TITLE
remove Jekyll toc

### PR DESCRIPTION
**Pull request by JingGe**
Remove all unused Jekyll {% toc %}

### CODE REVIEW

**Comment by JingGe** [comment]
@MartijnVisser Would you like to check if I did it right? Thanks! Just saw weird `{% toc %}` while reading Flink blogs, e.g. https://flink.apache.org/2022/02/22/scala-free-in-one-fifteen/

**Review by zentol — CHANGES_REQUESTED** [review]
the html files are generated and changes should be made to the .md files.

**Comment by JingGe** [comment]
Thanks @zentol for the hint. I have addressed it. Please check again.

File: docs/content/posts/2019-08-22-release-1.9.0.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ lists](https://flink.apache.org/community.html#mailing-lists) or
 feedback is very much appreciated!
 
 
-{% toc %}
+
```
Comment (zentol) [inline docs/content/posts/2019-08-22-release-1.9.0.md:45]: can you remove this line instead? There isn't really a reason to replace the toc with an empty line in the .md files (even if they dont get rendered)

Comment (JingGe) [reply docs/content/posts/2019-08-22-release-1.9.0.md:45]: sure, fixed.

**Pull request merged**